main.py

```diff
@@ -1,10 +1,30 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as player_file:
+        player_data = json.load(player_file)
+    for player_name in player_data.keys():
+        info = player_data[player_name]
+        email = info["email"]
+        bio = info["bio"]
+        guild = info["guild"]
+        guild_class = Guild.objects.get_or_create(name=guild["name"], description=guild["description"])
```

Comment:
The `get_or_create` method returns a tuple `(object, created)`. You should unpack this tuple to get the `Guild` instance. For example: `guild_class, created = Guild.objects.get_or_create(...)`.